changelog/change_add_schema_and_schemafile_to_default_include_list.md

```diff
@@ -0,0 +1 @@
+* [#11983](https://github.com/rubocop/rubocop/pull/11983): Add `*.schema` and `Schemafile` to default `Include` list. ([@ydah][])
```

Comment:
It might be user-friendly to mention that these are Ridgepole files.

Reply:
I updated this PR.